server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/EventDeadLettersRoutesTest.java

```diff
@@ -905,10 +923,11 @@ void postRedeliverSingleEventShouldFailWhenInvalidAction() {
 
         @Test
         void postRedeliverSingleEventShouldFailWhenMissingAction() {
-            deadLetters.store(new EventBusTestFixture.GroupA(), EVENT_1, INSERTION_ID_1).block();
-
+            InsertionId insertionId = deadLetters.store(new EventBusTestFixture.GroupA(), EVENT_1).block();
+            assertThat(insertionId).isNotNull();
+            
             when()
-                .post("/events/deadLetter/groups/" + SERIALIZED_GROUP_A + "/" + INSERTION_UUID_1)
+                .post("/events/deadLetter/groups/" + SERIALIZED_GROUP_A + "/" + insertionId.getId().toString())
```

Comment:
Don't we have a convenience `insertionId.asString()`?

Reply:
Actually we don't have this method but i will make it.